hack/windows/Uninstall-OVS.ps1

```diff
@@ -17,19 +17,24 @@ if ($DeleteBridges) {
     }
 }
 
-# Stop and delete ovs-vswitchd service
-stop-service ovs-vswitchd
-sc.exe delete ovs-vswitchd
+# Stop and delete ovs-vswitchd service if it exists
 if (Get-Service ovs-vswitchd -ErrorAction SilentlyContinue) {
```

Comment:
What is the result of command "Get-Service ovs-vswitchd -ErrorAction SilentlyContinue" if Windows service ovs-vswitchd does not exist, $true or $false?

Reply:
$false